scheduler/docs/configuration.asc

```diff
@@ -98,6 +98,13 @@ We'll look at the configurable options in turn:
   This sets the role that Cook will connect to Mesos with. Default: `*`
   You can omit this property unless you've enabled security features in Mesos. The value should be in authorized list for the current `:principal` in `register_frameworks` Action in Mesos Authorization file. See http://mesos.apache.org/documentation/latest/authorization/ for details.
 
+`:enable-gpu-support`::
```

Comment:
I'd prefer to just link to Kevin's GPU doc--there is literally nothing special about configuring GPUs for cook vs for any Mesos framework, since everything Cook-specific is auto-enabled with the option.